extension/framework/framework.go

```diff
@@ -81,13 +90,18 @@ func TestExtensions(context *cli.Context) {
 	}
 	if runJsExt {
 		testFiles := getTestFiles(context.Args(), "js")
-		if ret := RunTests(testFiles, context.Bool("verbose") || config != nil, context.String("run-test"), context.Int("parallel")); ret != 0 {
+		if ret := RunTests(testFiles, context.Bool(FlagVerbose) ||
+			config != nil, context.String(FlagRunTest), context.Int(FlagParallel)); ret != 0 {
 			os.Exit(ret)
 		}
 	}
 	if runSoExt {
 		testFiles := getTestFiles(context.Args(), "so")
-		if err := gorunner.NewTestRunner(testFiles, context.Bool("verbose") || config != nil, context.String("run-test"), context.Int("parallel")).Run(); err != nil {
+		printAllLogs := context.Bool(FlagVerbose) || config != nil
+		testFilter := context.String(FlagRunTest)
+		workers := context.Int(FlagParallel)
+		schemas := config.GetStringList("schemas", []string{})
```

Comment:
`config` is `nil` when tests are run without config flag and this panics.

Reply:
thanks, done